tcms/testruns/tests/test_admin.py

```diff
@@ -22,3 +28,20 @@ def test_regular_user_can_not_change_testrun(self):
         response = self.client.get(reverse('admin:testruns_testrun_change',
                                            args=[self.test_run.pk]))
         self.assertRedirects(response, reverse('testruns-get', args=[self.test_run.pk]))
+
+    def test_regular_user_can_not_delete_testrun(self):
+        response = self.client.get(reverse('admin:testruns_testrun_delete',
+                                           args=[self.test_run.pk]))
+        self.assertRaisesMessage(response, ('Permission denied: TestRun does not belong to you'))
```

Comment:
assertRaisesMessage is used when you want to verify that a method/function call raises an Exception with a particular error message string. It doesn't apply here. 

The error message is shown as a warning on the web page and it is part of the HTML. You already know a way how to verify what is contained in the HTML so use it.